### PR TITLE
PAN-2345: Add Sonic Blaze chainid

### PR DESCRIPTION
**Pull request by jacekv**
## What type of PR is this? (check all applicable)

- [ ] Refactor
- [ ] Feature
- [X] Bug Fix
- [ ] Optimization
- [ ] Documentation Update

## Description
Added Sonic Blaze chain id to scripts
## Related Tickets & Documents

- Related Issue #
- Closes #

## QA Instructions

_Please replace this line with instructions on how to test your changes._

## [optional] Are there any post deployment tasks we need to perform?

### CODE REVIEW

**Review by chandraprakash — DISMISSED** [review]

**Comment by jacekv** [comment]
@chandraprakash Sorry for making your approval stale - I added a parameter to the PantosForwarder initialization.

**Comment by markuslevonyak** [comment]
There are conflicts with my PR, which is already open since quite a while (but can't be merged due to a missing code owner approval):
https://github.com/pantos-io/ethereum-contracts/pull/75

File: script/DeployContracts.s.sol
Comment (markuslevonyak) [inline script/DeployContracts.s.sol:121]: This argument is not needed.

Comment (jacekv) [reply script/DeployContracts.s.sol]: Removed it :) Good catch

**Review by markuslevonyak — APPROVED** [review]

File: script/helpers/PantosBaseScript.s.sol
Comment (danut13) [reply script/helpers/PantosBaseScript.s.sol:125]: I think we should remove the old chain ID (64165) and keep only the new one, since we should operate on a single testnet at a time. Also, the mainnet chain ID is available so we should add it. 

However, this PR is created from a fork which we won't have access to. I will close this PR and create a new one.

**Comment by danut13** [comment]
https://github.com/pantos-io/ethereum-contracts/pull/81

**Pull request closed without merging**
